.github/action-inputs-config.json

```diff
@@ -20,28 +20,22 @@
     "validation_message": "Letters or - only"
   },
   {
-    "name": "create_release",
-    "validation_regex": "/^(true|false)$/.source",
```

Comment:
Can minimally add these back in as needed. I'm expecting yup bools to have some of this built-in? If not when adding UI elements, we can just add back in later